terraform/.opentofu-version

```diff
@@ -0,0 +1 @@
+1.8.7
```

Comment:
I don't think this file should be included in the repository.

Reply:
I removed the `.opentofu-version` file for `tenv` and removed the comments about it that I had added to the `README.md`.